todo_app/index/__init__.py

```diff
@@ -0,0 +1,4 @@
+from . import trello, objects
+from flask import Blueprint
+
+index = Blueprint('index', __name__)
```

Comment:
I don't think this is doing anything, as you import the blueprint from `views.py` instead

Reply:
yes, now removed!